src/app/core/mutations/subsscription.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from 'apollo-angular';
+
+export type GENERAL_FEED_SUB_MUTATION_RESPONCE = {
+    isSubscribed: boolean;
+    feedId: number;
+}
+
+
+
+export const GENERAL_FEED_SUB_MUTATION = gql<GENERAL_FEED_SUB_MUTATION_RESPONCE, {feedId: number , isSubscribed: boolean}>`
+mutation GENERAL_FEED_SUB_MUTATION($isSubscribed: Boolean!, $feedId: Float!) {
+    changeSubscription(isSubscribed: $isSubscribed, feedId: $feedId)
+  }
+`;
+
```

Comment:
typo in filename, add comment to mutation (what it does, input, output)

Reply:
fixed